src/Form/UserForm.php

```diff
@@ -1,227 +1,262 @@
         $inputFilter->add([
-                'name'     => 'email',
-                'required' => true,
-                'filters'  => [
-                    ['name' => 'StringTrim'],                    
-                ],                
-                'validators' => [
-                    [
-                        'name'    => 'StringLength',
-                        'options' => [
-                            'min' => 1,
-                            'max' => 128
-                        ],
+            'name' => 'email',
+            'required' => true,
+            'filters' => [
+                ['name' => 'StringTrim'],
+            ],
+            'validators' => [
+                [
+                    'name' => 'StringLength',
+                    'options' => [
+                        'min' => 1,
+                        'max' => 128
                     ],
-                    [
-                        'name' => 'EmailAddress',
-                        'options' => [
-                            'allow' => \Zend\Validator\Hostname::ALLOW_DNS,
-                            'useMxCheck'    => false,                            
-                        ],
+                ],
+                [
+                    'name' => 'EmailAddress',
+                    'options' => [
+                        'allow' => \Zend\Validator\Hostname::ALLOW_DNS,
+                        'useMxCheck' => false,
+                    ],
+                ],
+                [
+                    'name' => UserExistsValidator::class,
+                    'options' => [
+                        'entityManager' => $this->entityManager,
+                        'user' => $this->user
                     ],
-                    [
-                        'name' => UserExistsValidator::class,
-                        'options' => [
-                            'entityManager' => $this->entityManager,
-                            'user' => $this->user
-                        ],
-                    ],                    
                 ],
-            ]);     
-        
+            ],
+        ]);
+
         // Add input for "full_name" field
         $inputFilter->add([
-                'name'     => 'full_name',
+            'name' => 'full_name',
+            'required' => true,
+            'filters' => [
+                ['name' => 'StringTrim'],
+            ],
+            'validators' => [
+                [
+                    'name' => 'StringLength',
+                    'options' => [
+                        'min' => 1,
+                        'max' => 512
+                    ],
+                ],
+            ],
+        ]);
+
+        if ($this->scenario == 'create') {
+
+            // Add input for "password" field
+            $inputFilter->add([
+                'name' => 'password',
                 'required' => true,
-                'filters'  => [                    
-                    ['name' => 'StringTrim'],
```

Comment:
No, it seems that You have been mistaken by old filters and new filters location